pallets/stableswap/src/trade_execution.rs

```diff
@@ -322,6 +322,8 @@ where
 					Some(trade_fee),
 					&asset_pegs,
 				)
+				.ok_or_else(|| ExecutorError::Error(ArithmeticError::Overflow.into()))?
+				.reciprocal()
```

Comment:
wouldnt be be better to just swap the asset as input to the calculate spot price function ?

i believe it would be more clear - this reciprocal call is bit lost here imo